Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityEditorSettings.cs

```diff
@@ -296,6 +310,33 @@ private static void CheckOptimalConfiguration()
             }
         }
 
+        
+        /// <summary>
+        /// Helper method to update the a layer name at given index in the related project asset file
+        /// </summary>
+        /// <param name="layerNumber">index of layer to modify. Should be between 8 and 31</param>
+        /// <param name="layerName">Name of layer to set</param>
+        private static void CreateLayer(int layerNumber, string layerName)
+        {
+            SerializedObject tagManager = new SerializedObject(AssetDatabase.LoadAllAssetsAtPath("ProjectSettings/TagManager.asset")[0]);
+
+            SerializedProperty layers = tagManager.FindProperty("layers");
+            if (layers == null || !layers.isArray)
+            {
+                Debug.LogWarning("Can't set up the layers.  It's possible the format of the layers and tags data has changed in this version of Unity.");
```

Comment:
Nit: Consider adding a comment or something around here suggesting the user manually add the layer themselves with the given name + ID. At least that way they'd have a workaround if they hit this warning.